spec/index.bs

```diff
@@ -3292,6 +3292,7 @@ Issue(https://github.com/gpuweb/gpuweb/issues/693): add a limit to the number of
 
 <script type=idl>
 interface GPUCommandBuffer {
+    double getExecutionTime();
 };
 GPUCommandBuffer includes GPUObjectBase;
 </script>
```

Comment:
The definition of executionTime should be here instead of below. (`<dl dfn-type=attribute dfn-for=GPUCommandBuffer> ...`)

It can say something like `this.[[executionTimePromise]] if it is set; otherwise, a [=rejected promise|promise rejected with=] {{OperationError}}`.